### ISSUE TITLE
'Group Nodes' feature does not work with Anything Anywhere inputs, Queueing fails silently.

### ISSUE THREAD

**Issue by Chalice** [chrisgoringe/cg-use-everywhere#83]
How to replicate:
1) Open the default Comfy workflow
2) Create a new 'Anything Everywhere?' node
3) Give its regexp the value ^model$
4) link the default workflow's 'Load Checkpoint' node model to the 'Anything Anywhere?' node
5) Unlink the 'model' input from the default 'KSampler' node to make it use the 'Anything Anywhere?' model input
6) Select the 'KSampler' and the 'VAE Decode' nodes, rightclick -> 'Convert To Group Node'
7) Try to hit the Queue

Result:
Nothing gets queued. This always happens once you have Group Nodes in the workflow that use a remote Anything Everywhere input.

8) Rightlick the new Group Node -> 'Convert To Nodes'
9) Try to Queue

Result:
The workflow queues as normal.

**Comment by chrisgoringe** [comment]
Looks like an issue with an update to Comfy. High priority to fix (but I'm going to be off-grid for a few weeks in Dec/Jan...)

**Comment by chrisgoringe** [comment]
Additional notes: if you try to UE to the input of a Group Node, it fails silently with:

```
use_everywhere.js:34  Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'inputs')
    at use_everywhere.js:34:39
    at Array.forEach (<anonymous>)
    at use_everywhere.js:30:22
    at Array.forEach (<anonymous>)
    at analyse_graph (use_everywhere.js:29:51)
    at async ComfyApp.queuePrompt (app.js:1859:16)
    at async app.queuePrompt (nodeMenu.js:54:13)
```

If you try to use the output of a group, it just fails to connect.

In both cases, the "Show UE links" shows it as connected...

**Comment by chrisgoringe** [comment]
Partially fixed in 4.4.2

**Comment by chrisgoringe** [comment]
Added #84 to cover other cases
